integration-cli/docker_utils.go

```diff
@@ -458,6 +460,32 @@ func (d *Daemon) sock() string {
 	return fmt.Sprintf("unix://%s/docker.sock", d.folder)
 }
 
+func (d *Daemon) getBaseDeviceSize(c *check.C) int64 {
+
```

Comment:
I have changed the variable names to start with lowercase. The carriage returns are not random, they are in the output and needs to be trimmed. Otherwise the test fails.